### ISSUE TITLE
Support for sqlc

### ISSUE THREAD

**Issue by alg0s** [kyleconroy/sqlc#919]
Hi, I just wonder if sqlc is fully supporting SQLite yet? The github page seems to be changed from last time I visited and even the website has no indication of which databases it is currently supporting. 

Thanks.

**Comment by kyleconroy** [comment]
> even the website has no indication of which databases it is currently supporting.

Thanks for flagging this. I recently migrated to a separate documentation site and forgot to port the compatibility table.

SQLite support is planned, but no one is working on it right now. We're blocked on having a high-quality parser for SQLite's SQL dialect.

Duplicate of #161

**Comment by kyleconroy** [comment]
Added the table back here: https://docs.sqlc.dev/en/latest/reference/language-support.html
